db/migrate/20211109125417_update_cherrypicked_samples_biosero.rb

```diff
@@ -0,0 +1,77 @@
+# frozen_string_literal: true
+
+# Enable again Biosero events in view
```

Comment:
When it says "Enable **again**"... did the Biosero events used to be in the view?

Reply:
Yes, we accidentally rolled back this in last update I think

Reply:
ah ok!